tests/test_entity.py

```diff
@@ -38,6 +40,22 @@ def test_all_entities_refer_to_same_loader(self):
         for rule in evaluated_entity.rules:
             assert rule.loader == original_loader
 
+    def test_destination_to_dict_(self):
+
+        tpv_config = os.path.join(os.path.dirname(__file__), 'fixtures/mapping-rule-argument-based.yml')
+        loader = TPVConfigLoader.from_url_or_path(tpv_config)
+
+        # create a destination
+        destination = loader.destinations["k8s_environment"] #loader.destinations.values()[0]
+        # serialize the destination
+        serialized_destination = destination.to_dict()
+        # deserialize the same destination
+        deserialized_destination = Destination.from_dict(loader, serialized_destination)
+        # make sure the deserialized destination is the same as the original
+        self.assertEqual(deserialized_destination, destination)
+        assert deserialized_destination == destination
```

Comment:
The assertEqual above should do the same thing, so I think it's ok to remove this.
```suggestion
```